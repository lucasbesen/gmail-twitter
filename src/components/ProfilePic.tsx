import styled from 'styled-components';

const ProfilePic = styled.img.attrs({
  src:
    'https://lh3.googleusercontent.com/-biDP4iMaG_A/AAAAAAAAAAI/AAAAAAAAAAA/ACHi3rdNR-6OWw8tGnSp58jq9b4qx0FZVw.CMID/s64-c/photo.jpg',
})`
  width: 32px;
  height: 33.5px;
  border-radius: 15px;
`;

export default ProfilePic;
