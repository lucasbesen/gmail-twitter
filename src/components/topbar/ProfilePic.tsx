import React from 'react';
import styled from 'styled-components';

const Wrapper = styled.div`
  display: flex;
  align-items: center;
  justify-content: center;
  width: 42px;
  height: 43.5px;
  cursor: pointer;
  border-radius: 25px;

  :hover {
    background-color: #eee;
  }
`;

const Image = styled.img.attrs({
  src:
    'https://lh3.googleusercontent.com/-biDP4iMaG_A/AAAAAAAAAAI/AAAAAAAAAAA/ACHi3rdNR-6OWw8tGnSp58jq9b4qx0FZVw.CMID/s64-c/photo.jpg',
})`
  border-radius: 15px;
  width: 32px;
  height: 33.5px;
`;

const ProfilePic: React.FC = () => (
  <Wrapper>
    <Image />
  </Wrapper>
);

export default ProfilePic;
