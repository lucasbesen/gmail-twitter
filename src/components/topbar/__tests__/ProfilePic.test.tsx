import React from 'react';
import renderer from 'react-test-renderer';

import ProfilePic from '../ProfilePic';

it('should render ProfilePic without error', () => {
  const component = renderer.create(<ProfilePic />);
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
