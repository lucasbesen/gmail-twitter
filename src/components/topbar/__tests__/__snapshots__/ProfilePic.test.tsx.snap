// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ProfilePic without error 1`] = `
<div
  className="sc-bdVaJa kIbVTK"
>
  <img
    className="sc-bwzfXH bFYweS"
    src="https://lh3.googleusercontent.com/-biDP4iMaG_A/AAAAAAAAAAI/AAAAAAAAAAA/ACHi3rdNR-6OWw8tGnSp58jq9b4qx0FZVw.CMID/s64-c/photo.jpg"
  />
</div>
`;
